Crow/Router/Exceptions/RoutingLogicException.php

```diff
@@ -11,6 +11,7 @@ class RoutingLogicException extends LogicException
 {
     public function __construct(string $message = "", int $code = 0, Throwable $previous = null)
     {
+        //construct is definng a method parent of these variables
```

Comment:
Is this line really important ? Isn't it obvois that the parent method is accepting these vars :)